### DISCUSSION TITLE
Impossible to --shutdown a sandbox by PID

### DISCUSSION THREAD

**Discussion by tredondo** [netblue30/firejail#4096] [Q&A] [answered]
Question from https://firejail.wordpress.com/documentation-2/ per https://github.com/netblue30/firejail/issues/2713#issuecomment-798831498 (haven't checked if reproducible):

> After running `sudo firecfg` its impossible to shut down `firejail` using command `firejail --shutdown=PID` because the PID is everchanging. Each invocation of `firejail --list to get the PID will change the PID of the result (e.g 4054: --list)

**Comment by SkewedZeppelin** [comment, marked as answer]
The solution to this is setting names in each profile's override.

```
man firejail:
	--name=name
	      Set sandbox name. Several options, such as  --join  and  --shut‐
	      down, can use this name to identify a sandbox.

man firejail-profile:
	name sandboxname
	      Set sandbox name.
```

Off-topic: It is absolutely frustrating how broken markdown from emails is. You can't even add markdown by editing afterwards. Why Github? Why?
